<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiDataVaultItem</class>
 <widget class="QWidget" name="UiDataVaultItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>110</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutDataVaultItem">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTitle">
     <item>
      <widget class="QLabel" name="labelDataVaultTitel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="indent">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDataVaultMenu">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../../res/olbaflinx.qrc">
         <normaloff>:/datavault/menu</normaloff>:/datavault/menu</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDataVaultPassword">
     <item>
      <spacer name="horizontalSpacerDataVaultPasswordLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDataVaultPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDataVaultOpen">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../../res/olbaflinx.qrc">
         <normaloff>:/datavault/open</normaloff>:/datavault/open</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDataVaultPasswordRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelDataVaultFilePath">
     <property name="text">
      <string>-</string>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditDataVaultPassword</tabstop>
  <tabstop>pushButtonDataVaultMenu</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/olbaflinx.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonDataVaultMenu</sender>
   <signal>clicked()</signal>
   <receiver>UiDataVaultItem</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDataVaultOpen</sender>
   <signal>clicked()</signal>
   <receiver>UiDataVaultItem</receiver>
   <slot>openVault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showMenu()</slot>
  <slot>openVault()</slot>
 </slots>
</ui>
